<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtModbus</class>
 <widget class="QMainWindow" name="QtModbus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtModbus</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>22</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStart_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start_2</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="PositionReadButton">
       <property name="text">
        <string>Read Position</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="PositionEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>321</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="4">
      <widget class="QPushButton" name="PointSetButton">
       <property name="text">
        <string>Set point</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="Point2Edit"/>
     </item>
     <item row="2" column="2" colspan="3">
      <widget class="QPushButton" name="FirmnessSetButton">
       <property name="text">
        <string>Set firmness</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="Point1Edit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Hz</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="FirmnessEdit"/>
     </item>
     <item row="3" column="2" colspan="3">
      <widget class="QPushButton" name="VibFrequencySetButton">
       <property name="text">
        <string>Set Vib_Frequency</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="VibFrequencyEdit"/>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="ModeEdit"/>
     </item>
     <item row="4" column="2" colspan="3">
      <widget class="QPushButton" name="btnStopVibration">
       <property name="text">
        <string>stop vibration</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QPushButton" name="btnVibration">
       <property name="text">
        <string>vibration</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="ModeSetButton">
       <property name="text">
        <string>Set Mode</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="PositionlistWidget">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>140</y>
      <width>131</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="AutoReadPositioncheckBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>自动读位置</string>
    </property>
   </widget>
   <widget class="QListWidget" name="SwitchlistWidget">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>140</y>
      <width>131</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="AutoReadSwitchcheckBox">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>120</y>
      <width>181</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>自动读开关</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="readEdit">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="readButton">
         <property name="text">
          <string>Read</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="writeEdit">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="writeButton">
         <property name="text">
          <string>Write</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnStart_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>// Start_1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="FlipTestcheckBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>280</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Pedal flip</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>712</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
